### ISSUE TITLE
Some content not rendering in Safari, but will render on window resize

### ISSUE THREAD

**Issue by kitsandkats** [google/material-design-lite#4741]
When I load a page that is using the MDL javascript, text content below the fold will not display. However, it will display when that text is highlighted or if the window is resized. It works fine for me in Chrome and Firefox. And when I remove the link to the MDL js file, the issue goes away.

The bug can be seen [here](https://kitsandkats.github.io/lcl-brazil/events.html).

I'm using MDL version 1.2.1. Safari 9.1.3 and OS X El Capitan 10.11.6.

**Comment by Garbee** [comment]
This sounds similar to #4354. With the new data this adds about the MDL JS being removed and it works, it seems related to how we handle the layout by creating a new container node on-the-fly and appending the content as a child of that.

Not sure if there is anything we can really do here internally as the Layout is complicated and developers are already working around the quirks in production. Changing any of that could be a big breaking change.

@sgomes Any thoughts here?

**Comment by Garbee** [comment]
Yup, a regression in Safari's rendering happened somewhere. But tracking that down is extremely difficult. Which is key to finding a proper solution to the problem.

**Comment by SabineMa** [comment]
I have co come back to this issue here which is the same as #4354 . I was using 
```
.mdl-card {  overflow: visible}
```
And it worked fine, -> page was rendered in safari.

But one of my card views is small and I needed this solution, too: 
#1407 #943 
(overlapping menues in cards)

```
.mdl-card {   overflow: visible;    z-index: auto;}
```

But that removes the solution for this issue. page is not rendered in safati!
Both solutions are in conflict with each other.
Does anyone have the same problem and a solution?

**Comment by sgomes** [comment]
Sorry for only getting to this issue now! @SabineMa, could you confirm this is still happening with 1.3.0?

**Comment by SabineMa** [comment]
yes, but in my case only theoretically because I assigned two different css definitions. So, for me the problem is fixed by this workaround.
